examples/tutorials/remote_example.py

```diff
@@ -59,19 +59,24 @@ async def run_batch(adder_client, send_tensor=False):
 
 
 def main():
-    adder = Adder()
+    adder1 = Adder('adder1')
```

Comment:
I'd suggest not to change this file because it is used for most basic tutorials.